template-tags.php

```diff
@@ -697,7 +709,7 @@ function coauthors_get_avatar( $coauthor, $size = 32, $default = '', $alt = fals
 
 	// Make sure we're dealing with an object for which we can retrieve an email
 	if ( isset( $coauthor->user_email ) ) {
-		return get_avatar( $coauthor->user_email, $size, $default, $alt );
+		return get_avatar( $coauthor->user_email, $size, $default, $alt, array( 'class' => $class ) );
```

Comment:
I'm seeing that for `get_avatar()`, you can pass the `$class` in as an array, so would it be necessary to `implode()` it on line 699? https://codex.wordpress.org/Function_Reference/get_avatar Related to comment above ^^

Reply:
The value in `$args['class']` for `get_avatar()` can be an array or a string. So I don't think we need to `implode()`.
https://github.com/WordPress/WordPress/blob/master/wp-includes/pluggable.php#L2533

However, I like your previous suggestion of moving the check on line 698 to `get_guest_author_thumbnail()`. That way, there will be minimal changes to this function and the logic for differentiating between an array and string will be the responsibility of  `get_guest_author_thumbnail` and `get_avatar()`